pyaedt/emit.py

```diff
@@ -487,7 +487,7 @@ def interaction_domain():
         Returns
         -------
         :class:`Emit.InteractionDomain`
-            Domain used to define a set of interacting interferers and receivers
+            Domain used to define a set of interacting interferers and receivers.
```

Comment:
```suggestion
            Provides the domain used to define a set of interacting interferers and receivers.
```
FYI: The style is that class descriptions start with a verb ending in an "s".